frontend/modules/viewpoints/viewpoints.sagas.ts

```diff
@@ -214,17 +214,16 @@ export function* showViewpoint({teamspace, modelId, view, ignoreCamera}) {
 			yield Viewer.isViewerReady();
 
 			const viewpoint = view.viewpoint;
-			if (!viewpoint) {
```

Comment:
if theres' no viewpoint there's nothing to do?

I don't think it's possible to have clipping planes but no view. (esp after the refactoring with views/issues/risks.)

Any reason why this was changed?

Reply:
reason I ask is because I remember putting this in to plug a white screen problem. so I'm not sure if we're recreating it again.